terraform-vpc/securitygroup.tf

```diff
@@ -0,0 +1,97 @@
+resource "aws_security_group" "website" {
+    name = "website"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "website"
+    }
+
+    ingress {
+        from_port = 22
+        to_port = 22
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 80
+        to_port = 80
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 443
+        to_port = 443
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
```

Comment:
Where will this security group be used?
Why would you open those 3 ports(22, 80 and 443) for any component of the solution?

Reply:
My perception initially was 443 / 80 needed to be open to allow access to the WP Containers.
But we can set it from the docker run command instead.

As for 22, I was under the impression that this should be open for Root-Cause-Analysis or troubleshooting in case there was an issue with one of the ECS Containers

Thoughts?

Reply:
containers will be exposed on port 80(unless you guys add a certificate in the containers to have it running on port 443.
even that containers will expose 80, on the host, this port 80 will be something different(pretty much random).
ALB will be listening on 443
ECS hosts(EC2) can be accessed by 22...

So you pretty much created a single Security group for several different kind of access...

My suggestion is to only create resources for things that will be implemented in this PR.
If you're not implementing ALB, dont include SG for 443 connections.

If you're not implementing containers, don't include SG for accessing the containers.

The reason for that is that you guys should be all collaborating to this code. It's fine that you know that this SG will be used in a future PR, but someone else doing this future PR, might create a duplicated resource and you'll guys will end up having more resources created than required.

Reply:
Thank you Kiko,
I will remove the SG from this PR.
And address it in a separate PR